packages/massa-proto-cli/src/protobuf.ts

```diff
@@ -57,8 +57,8 @@ export async function getProtoFunction(protoPath: string): Promise<ProtoFile> {
   const messageNames = Object.keys(protoJSON.nested);
 
   // check if the proto file contains 2 messages
-  if (messageNames.length !== 2) {
-    throw new Error('Error: the protoFile should contain 2 messages.');
+  if (messageNames.length > 2) {
+    throw new Error('Error: the protoFile should contain maximum 2 messages.');
```

Comment:
I saw the same change in an other pr. I think we should keep only this one but I'll wait for your confirmation

Reply:
idk which pr implement the same change but we can keep this one